ProcessLib/ProcessVariable.h

```diff
@@ -87,14 +90,20 @@ class ProcessVariable
 		}
 	}
 
-	double getInitialConditionValue(MeshLib::Node const& n) const
+	double getInitialConditionValue(MeshLib::Node const& n,
+	                                int const component_id) const
 	{
-		return _initial_condition->getValue(n);
+		return _initial_condition->getValue(n, component_id);
 	}
 
+	// Get or create a property vector for results.
+	// The returned mesh property size is number of mesh nodes times tuple size.
+	MeshLib::PropertyVector<double>& getOrCreateMeshProperty();
+
 private:
 	std::string const _name;
-	MeshLib::Mesh const& _mesh;
+	MeshLib::Mesh& _mesh;
+	int _tuple_size;
```

Comment:
I currently don't see, why the tests fail (no access to Jenkins). From my side only minor things. Apart from them :+1:
Have a nice weekend.